mycodo/mycodo_flask/utils/utils_trigger.py

```diff
@@ -172,13 +172,13 @@ def trigger_activate(trigger_id):
     actions = Actions.query.filter(
         Actions.function_id == trigger_id)
 
-    if not actions.count():
+    if not actions.count() and mod_trigger.trigger_type != 'trigger_run_pwm_method':
```

Comment:
Allow to activate a run_pwm_method trigger even if there are no actions assigned. Reason: it sets the PWM output regardless of the actions. Users might only want this. Actions are still possible.